base-containers/base/inginious/feedback.py

```diff
@@ -86,6 +86,27 @@ def set_problem_feedback(feedback, problem_id, append=False):
     save_feedback(rdict)
 
 
+def set_tag(tag, value):
```

Comment:
Maybe we should merge this completely with ``tests``, by removing the ``inginious.tests`` module. Moreover, as those values are now always returned and not only for debug, putting them in a separate file is no more useful. You can merge it directly in the feedback dictionary. You'll have something like : 

    rdict = load_feedback()
    tests = rdict.setdefault("tests", {})
    tests[tag] = value == True
    save_feedback(rdict)